setup.py

```diff
@@ -35,7 +35,7 @@ def get_long_description():
     license='MIT',
     url='https://github.com/albu/albumentations',
     packages=find_packages(exclude=['tests']),
-    install_requires=['numpy>=1.11.1', 'scipy', 'opencv-python-headless', 'imgaug>=0.2.5,<0.2.7', 'PyYAML'],
+    install_requires=['numpy>=1.11.1', 'scipy', 'opencv-python>4.1.0', 'imgaug>=0.2.5,<0.2.7', 'PyYAML'],
```

Comment:
Are there any problems with OpenCV older than 4.1.0 or it is chosen to be a recent enough version?

Reply:
Bumped to most recent version

Reply:
I reverted back versionless `opencv-python-headless` to have more clear changeset.